eng/Tools.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="$(RepositoryEngineeringDir)VmrLayout.props" />
+
+  <!-- Override RestoreSources in Source-Build builds so the Arcade SDK restores its packages from offline sources. -->
+
+  <PropertyGroup>
+    <RestoreSources Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(IsTestRun)' != 'true'">$(PrebuiltPackagesPath);$(PreviouslySourceBuiltPackagesPath)</RestoreSources>
+  </PropertyGroup>
```

Comment:
@jkoritzinsky I'm still not exactly sure why this was needed. What changed that now requires these feeds to be used when restoring Arcade's Tools.proj?

Reply:
Why wasn't that necessary before?

Reply:
Before this change we weren't using `Tools.proj` in the orchestrator as we weren't using the Arcade build infrastructure (we were directly invoking msbuild on the root `build.proj`, not building using the bootstrapped Arcade's `Build.proj`)